core/Updates/4.0.0-rc4.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Config;
+use Piwik\DataAccess\ArchiveTableCreator;
+use Piwik\Date;
+use Piwik\Updater;
+use Piwik\Updates as PiwikUpdates;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+
+/**
+ * Update for version 4.0.0-rc4.
+ */
+class Updates_4_0_0_rc4 extends PiwikUpdates
+{
+    /**
+     * @var MigrationFactory
+     */
+    private $migration;
+
+    public function __construct(MigrationFactory $factory)
+    {
+        $this->migration = $factory;
+    }
+
+    public function getMigrations(Updater $updater)
+    {
+        $migrations = [];
+
+        $dates = ['2020-01-01', '2020-10-01', '2020-09-01'];
```

Comment:
@diosmosis does it need november in there too? I reckon we could actually remove September but no preference. We only released the beta on like 29th of September so it wouldn't have been much of an issue back then

Reply:
oh yes, it should, will fix

Reply:
@tsteur updated